src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1

```diff
@@ -0,0 +1,162 @@
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        ${DefaultProfile},
+
+        [Parameter(DontShow, HelpMessage = 'Wait for .NET debugger to attach.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        ${Break},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline.
+        ${HttpPipelineAppend},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline.
+        ${HttpPipelinePrepend},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use.
+        ${Proxy},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call.
+        ${ProxyCredential},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy.
+        ${ProxyUseDefaultCredentials}
+    )
+
+    process {
+        if (!(Get-Module -ListAvailable -Name SimplySQL)){
+            Write-Error "This cmdlet requires SimplySQL module. Please install the module first by running Install-Module -Name SimplySQL."
+        }
+        Import-Module SimplySQL
+        
+        $AdministratorUserName = [string]::Empty
+        if ($PSBoundParameters.ContainsKey('AdministratorUserName')) {
+            $AdministratorUserName = $PSBoundParameters.AdministratorUserName
+            $null = $PSBoundParameters.Remove('AdministratorUserName')
+        }
+
+        $DatabaseName = 'flexibleserverdb'
+        if ($PSBoundParameters.ContainsKey('DatabaseName')) {
+            $DatabaseName = $PSBoundParameters.DatabaseName
+            $null = $PSBoundParameters.Remove('DatabaseName')
+        }
+
+        $Query = [string]::Empty
+        if ($PSBoundParameters.ContainsKey('QueryText')) {
+            $Query = $PSBoundParameters.QueryText
+            $null = $PSBoundParameters.Remove('QueryText')
+        }
+
+        $Password = . "$PSScriptRoot/../utils/Unprotect-SecureString.ps1" $PSBoundParameters['AdministratorLoginPassword']
```

Comment:
autorest generator will generate password to secure string directly.

Reply:
Will make the change. Thanks.